esm/html/script-element.js

```diff
@@ -1,18 +1,96 @@
-import {registerHTMLClass} from '../shared/register-html-class.js';
+import { booleanAttribute, stringAttribute } from '../shared/attributes.js'
+import { registerHTMLClass } from '../shared/register-html-class.js'
 
-import {TextElement} from './text-element.js';
+import { TextElement } from './text-element.js'
 
-const tagName = 'script';
+const tagName = 'script'
 
 /**
  * @implements globalThis.HTMLScriptElement
  */
 class HTMLScriptElement extends TextElement {
   constructor(ownerDocument, localName = tagName) {
-    super(ownerDocument, localName);
+    super(ownerDocument, localName)
```

Comment:
it'd be weird to have a single file with no semicolons and I think the linter would complain too ... so please use semicolons **everywhere** thank you!